beets/util/artresizer.py

```diff
@@ -557,13 +610,13 @@ def proxy_url(self, maxwidth, url, quality=0):
             return resize_url(url, maxwidth, quality)
 
     @property
-    def local(self):
+    def local(self) -> bool:
         """A boolean indicating whether the resizing method is performed
         locally (i.e., PIL or ImageMagick).
         """
         return self.local_method is not None
 
-    def get_size(self, path_in):
+    def get_size(self, path_in: AnyStr) -> Union[Tuple[int, int], AnyStr]:
```

Comment:
Can you correct this method to return `None` in the `else` branch? What's currently going on is just wrong (probably due to my own fault... the git blame is somewhat confusing, not sure what happened. Maybe a botched merge? copy&paste bug?)

Reply:
I don't think there's a way to conditionally return a single type. I can add it to be `Optional` though so it recognises that a null value is possible.

Reply:
Sorry, what I meant is that this should return `Optional[Tuple[int, int]]`, and in the else case `return None` instead of `return path_in`.